lib/improver/tests/utilities/test_load.py

```diff
@@ -148,17 +148,17 @@ def test_ordering_for_realization_coordinate(self):
         self.assertArrayAlmostEqual(result.coord_dims("latitude")[0], 1)
         self.assertArrayAlmostEqual(result.coord_dims("longitude")[0], 2)
 
-    def test_ordering_for_percentile_over_coordinate(self):
+    def test_ordering_for_percentile_coordinate(self):
         """Test that the cube has been reordered, if it is originally in an
-        undesirable order and the cube contains a "percentile_over"
+        undesirable order and the cube contains a "percentile"
```

Comment:
Nit-picking but can you remove the line break here please.

Reply:
Done.